### PR TITLE
feat: tinyproxy container

### PR DESCRIPTION
**Pull request by DerekRoberts**

### CODE REVIEW

**Review by ahbarnz-g — DISMISSED** [review]

File: podman/tinyproxy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	./configure && \
+	make && \
+	make install && \
+	cd .. && \
+	rm -rf ${tpRelease} ${tpRelease}.tar.gz && \
+	microdnf remove gcc make gzip tar wget && \
+	microdnf clean all
+
+# Configuration
+COPY ./conf/ /usr/local/etc/tinyproxy/
```
Comment (mbystedt) [inline podman/tinyproxy/Dockerfile:24]: Not a volume?

File: podman/tinyproxy/dev.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+docker build . -t tp
+docker run -p 23128:8888 --name=tp -v $(pwd)/conf/:/usr/local/etc/tinyproxy/:z tp
```
Comment (mbystedt) [inline podman/tinyproxy/dev.sh:2]: Getting mixed messages if this is hardcoded in the image or a volume...

File: podman/tinyproxy/dev.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+docker rm -fv tp
```
Comment (mbystedt) [inline podman/tinyproxy/dev.sh:3]: docker run --rm -p 23128:8888 --name=tp -v $(pwd)/conf/:/usr/local/etc/tinyproxy/:z tp

^ Why not just this?

**Review by mbystedt — COMMENTED** [review]
No show stoppers

File: podman/tinyproxy/Dockerfile (hunk 1 above)
Comment (DerekRoberts) [reply podman/tinyproxy/Dockerfile:24]: Sort of!  This is default image config.  Volume overrides.

File: podman/tinyproxy/dev.sh (hunk 2 above)
Comment (DerekRoberts) [reply podman/tinyproxy/dev.sh:2]: Both!  @ahbarnz-g has some use cases in mind.

File: podman/tinyproxy/dev.sh (hunk 3 above)
Comment (DerekRoberts) [reply podman/tinyproxy/dev.sh:3]: <facepalm>  ...thanks.  :)

**Review by ahbarnz-g — APPROVED** [review]

**Pull request merged**